relay_test.go

```diff
@@ -1361,3 +1362,207 @@ func inspectFrames(rh *relaytest.StubRelayHost) chan relay.CallFrame {
 	})
 	return frameCh
 }
+
+func TestRelayModifyArg2(t *testing.T) {
+	// largeKey := testutils.RandString(378)
```

Comment:
let's add a TODO here as well so it doesn't look like an accidental debug change

Reply:
actually on second thought, can we just remove the large key/val code altogether? I already have it in my branch so I can add it back in a follow-up diff. leaving commented code around like this feels weird.

Reply:
Yeah, I'll delete this, and just leave the "small modification" as an example with a comment, and delete the rest.